01_05_Conditions.txt

```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
-one stack. 
+    If Burning is not extinguished for two Rounds, the Burn intensifies by one stack. 
 
     Some burning chemicals, like white phosphorous or most napalms, contain the 
 oxygen that they need to burn and aren't extinguished with water or rolling 
 around. These sources of the Burning Condition will explicitly state that they 
-can't be put out with normal means and will need to be neutralized by other 
+can't be extinquished via normal means and will need to be neutralized by other 
 means, such as the Doctor's Purge ability.
 
 == Downed ==
 
-    Downed is a state that a character enteres when their health has fallen 
+    Downed is a state that a character enters when their health has fallen 
 below 0. See the "Death and Dying" section in the Combat Rules Document.
 
 == Disrupted ==
 
-    Certain Characters have the ability to disrupt their enemies either with
+    Certain characters have the ability to disrupt their enemies either with
 signal jamming, malware or sometimes direct electrical interference. When this
 succeeds, it prevents their target from using their more exotic talents.
 
-    A Character that is Disrupted cannot use Active Feats, Doctor Procedures, 
+    A character that is Disrupted cannot use Active Feats, Doctor Procedures, 
 Engineer Processes or Hacker Exploits. Many similar abilities are also prone to 
 Disruption. If they are not on this list, consult your GM. 
 
 == Frozen ==
 
-    Most Characters respond poorly to the application of extremely low 
+    Most characters respond poorly to the application of extremely low 
 temperatures to their anatomy. Biological targets tend to have the water in 
 their cells solidify into sharp ice in a process known as "Frostbite." 
 Cybernetic targets on the other hand tend to have their moving parts contract
 and wear down while their electrolytic fluid hardens. 
 
-    A Character that is Frozen takes d10 damage per round at the beginning of 
+    A character that is Frozen takes 1d10 damage per round at the beginning of 
 their turn. The Condition lasts until a source of heat is applied to the wound, 
 or with Medicine (Dex) Skill check, DC 50.
 
-== Fear ==
+== Fear (Afraid) ==
 
-    Both cybernetic and biologic Characters can react to the mental instinct for
+    Both cybernetic and biologic characters can react to the mental instinct for
 self-preservation. This usually happens when presented with the appearance of
 certain, agonizing death, though there are certainly other scary things in the
 universe.
 
-    A Character that has the Fear Condition may take no actions, except moving 
-away from the source of fear, or taking an action to roll to defeat fear. DC 50, 
-Will Save.
+    A character that is Afraid may take no actions, except moving away from
+the source of fear, or taking an action to roll a DC 50 Will Save to defeat Fear.
 
-== Fear, Greater ==
+== Fear, Greater (Afraid) ==
```

Comment:
Deeply Afraid?
Actually, as it stands it's consistent with Buzzed/Hammered (Intoxicated)

Reply:
That's why I stuck it in there :)